src/pages/docs/getting-started/index.md

```diff
@@ -0,0 +1,25 @@
+---
+date: "2015-03-10"
+title: "Getting started"
+description: "Getting started with FabricJS"
+---
+
```

Comment:
> [What is the frontmatter?](#issuecomment-647430704)

The meta data we store in the head section of a MD file.

For sorting the guide-pages in proper order, we rely on the **date** set appropriately. Perhaps later we'll use a JSON file to maintain the ordering